specs/agents/mobile/session.md

```diff
@@ -0,0 +1,18 @@
+# Session 
+
+### Overview
+A `session` is a collection of `events`, `logs`, and `spans` associated with a specific device within a specific period of time. 
+A `session` is represented by a unique identify that is attached to `event`, `logs`, and `spans` as an attribute. 
+
+The primary purpose of `sessions` are to provide insight into the series of user actions or events that lead up to a critical error or crash. Sessions also provide a means to quantify application usage.
+
+### How a session operates
+- All `events`, `logs`, and `spans` will have a `session` identifier attached as an attribute using the name `session.id`.
```

Comment:
Would `transaction` also have session info?

Reply:
certainly, I often forget about transactions since we operate on top of OpenTelemetry which only has the concepts of spans.